lib/einhorn/command.rb

```diff
@@ -146,11 +148,12 @@ def self.signal_all(signal, children=nil, record=true)
             spec[:signaled].add('KILL')
           end
         end
-      end
 
-      "Successfully sent #{signal}s to #{signaled.length} processes: #{signaled.keys}"
+        "Successfully sent #{signal}s to #{signaled.length} processes: #{signaled.keys}"
```

Comment:
Wait, lol, was that meant to be an `Einhorn.log_debug`? The call sites I see don't do anything with that string.

Reply:
Probably. Good catch, will add that.